### PR TITLE
Upgrades for tagging

### PR DESCRIPTION
**Pull request by CameronRP**
Added some improvements for tagging requested from issue #23

### CODE REVIEW

File: public/javascripts/viewRecording.js [hunk 1]
```diff
@@ -167,3 +177,42 @@ function setEndTimeAsCurrentTime() {
...
+    url: recordingsApiUrl + '/' + id,
+    type: 'DELETE',
+    headers: headers,
+    success: function(res) {
+      console.log(res);
+      window.alert("Deleted recording.")
+    },
+    error: function(err) {
+      console.log(err);
+      window.alert("Error with deleting recording.");
```
Comment (mjs) [inline public/javascripts/viewRecording.js:194]: Not for now, but we should probably use something less intrusive than `window.alert()` for telling the user that something happened.

File: views/includes/tags.pug [hunk 2]
```diff
@@ -18,6 +18,8 @@ form.form-horizontal#tagForm(onsubmit="return false")
       option hare
       option human
       option other
```
Comment (mjs) [inline views/includes/tags.pug]: wouldn't it make more sense to have `other` as second last?

**Review by mjs — APPROVED** [review]
So good to have all this done.

File: public/javascripts/viewRecording.js (hunk 1 above)
Comment (CameronRP) [reply public/javascripts/viewRecording.js:194]: Yep, this is just the easiest way for now.

File: views/includes/tags.pug (hunk 2 above)
Comment (CameronRP) [reply views/includes/tags.pug]: Yep, I'll change that.

**Pull request merged**